pkg/api/common.go

```diff
@@ -82,6 +82,20 @@ func (a *Auth) Validate() error {
 	return nil
 }
 
+type CommaSeparated []string
+
+func (v *CommaSeparated) String() string {
+	if v != nil {
+		return strings.Join(*v, ",")
+	}
+	return ""
+}
+
+func (v *CommaSeparated) Set(s string) error {
+	*v = strings.Split(s, ",")
+	return nil
+}
```

Comment:
```suggestion
type CommaSeparated map[string]struct{}

// I doubt this is necessary though
func (v *CommaSeparated) String() string {
	if v != nil {
		keys := make([]int, 0, len(v))
		for k := range v {
			keys = append(keys, k)
		}
		return strings.Join(keys, ",")
	}
	return ""
}

func (v *CommaSeparated) Set(s string) error {
	elem := strings.Split(s, ",")
	for _, s := range elem {  
		*v[s] = struct{}{}
	}
	return nil
}
```

I was hoping we could do something along those lines and remove the need to duplicate `PromscaleEnableFeatures` and `PromscaleEnabledFeaturesMap`

Parsing to the following type removes the need to have PromscaleEnabledFeaturesMap and PromscaleEnableFeatures and allows to proceed t